grpc/testing/messages.proto

```diff
@@ -31,6 +33,12 @@ message BoolValue {
 enum PayloadType {
   // Compressable text format.
   COMPRESSABLE = 0;
+
+  // Uncompressable binary format.
+  UNCOMPRESSABLE = 1;
```

Comment:
These were removed in C in https://github.com/grpc/grpc/pull/6839/files#r66543026

The DEPRECATED comments were only recently deleted in https://github.com/grpc/grpc/pull/15711 . We could finish the deprecation, since the enum is useless with only one entry.